net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/bgp.volt

```diff
@@ -103,16 +113,68 @@ POSSIBILITY OF SUCH DAMAGE.
         </table>
     </div>
 
-    <div id="routemaps" class="tab-pane fade in">
-        <table id="grid-routemaps" class="table table-responsive" data-editDialog="DialogEditBGPRouteMaps">
+    <div id="prefixlists" class="tab-pane fade in">
+        <table id="grid-prefixlists" class="table table-responsive" data-editDialog="DialogEditBGPPrefixLists">
+            <thead>
+                <tr>
+                    <th data-column-id="enabled" data-type="string" data-formatter="rowtoggle" data-sortable="false">{{ lang._('Enabled') }}</th>
+                    <th data-column-id="name" data-type="string" data-visible="true" data-sortable="true">{{ lang._('Name') }}</th>
+                    <th data-column-id="seqnumber" data-type="string" data-visible="true" data-sortable="true">{{ lang._('Secquence Number') }}</th>
+                    <th data-column-id="action" data-type="string" data-visible="true" data-sortable="false">{{ lang._('Action') }}</th>
+                    <th data-column-id="network" data-type="string" data-visible="true" data-sortable="false">{{ lang._('Network') }}</th>
+                    <th data-column-id="uuid" data-type="string" data-identifier="true" data-visible="false">{{ lang._('ID') }}</th>
+                    <th data-column-id="commands" data-formatter="commands" data-sortable="false">{{ lang._('Commands') }}</th>
+                </tr>
+            </thead>
+            <tbody>
+            </tbody>
+            <tfoot>
+                <tr>
+                    <td colspan="5"></td>
+                <td>
+                    <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
+                    <!-- <button data-action="deleteSelected" type="button" class="btn btn-xs btn-default"><span class="fa fa-trash-o"></span></button> -->
+                    </td>
+                </tr>
+            </tfoot>
+        </table>
+    </div>                
+
+    <div id="routemap-names" class="tab-pane fade in">
+        <table id="grid-routemap-names" class="table table-responsive" data-editDialog="DialogEditBGPRouteMapsName">
+            <thead>
+                <tr>
+                    <th data-column-id="enabled" data-type="string" data-formatter="rowtoggle">{{ lang._('Enabled') }}</th>
+                    <th data-column-id="name" data-type="string" data-visible="true">{{ lang._('Name') }}</th>
+                    <th data-column-id="linkedmatch" data-type="string" data-visible="true">{{ lang._('Match') }}</th>
+                    <th data-column-id="linkedset" data-type="string" data-visible="true">{{ lang._('Set') }}</th>
```

Comment:
Hm, in my understanding "Set" in context to BGP or Route-Maps is not translateable. 
What about "Set/Match value"? Otherwise I have to put it in quotation marks. 

I'd also not translate Match, you could translate it to Treffer, but for an engineer it breaks the understanding of route-maps completely.